public/js/review-main.js

```diff
@@ -0,0 +1,53 @@
+import { _, getData } from './util/util.js';
+import { $searchBar, $searchBarInput, $searchSuggestions, $rollingKeywords } from './util/ref.js';
+import { SearchBar, Rolling, KeywordSuggestion, AutoComplete } from './searchBar.js';
+import { URL, KEYCODE } from './util/data.js';
+import { parseRecommendedList } from './util/parser.js';
+const { log } = console;
+
+
+// search bar
+const searchBarArgs = {
+  domElem: $searchBar,
+  inputArea: $searchBarInput,
+  suggestions: $searchSuggestions,
+  rollingKeywords: $rollingKeywords
+};
+const searchBar = new SearchBar(searchBarArgs);
+searchBar.registerEvent();
+
+// rolling & search keyword suggestion
+async function initSuggestion() {
+  const recommendJsonData = await getData(URL.RECOMMEND);
+  
+  // rolling
+  const { list: recommendListInfo } = recommendJsonData;
```

Comment:
가능하면 불필요한 변수 할당을 최소화 하는 것이 좋은 것 같아요. 이름 짓기도 힘들잖아요 😭 
```suggestion
  const { list: recommendListInfo } = await getData(URL.RECOMMEND);
```

Reply:
대박,, 아예 인지하지 못하고 있었던 부분인데 정말 감삼다ㅠㅠㅠㅠㅠㅠㅜㅜㅜ